# CFG-RATE (0x06 0x08)
# Navigation/Measurement Rate Settings
#
# This message allows the user to alter the rate at which navigation solutions 
# (and the measurements that they depend on) are generated by the receiver. The 
# calculation of the navigation solution will always be aligned to the top of a 
# second zero (first second of the week) of the configured reference time 
# system. For protocol version 18 and later the navigation period is an integer 
# multiple of the measurement period.
#  - Each measurement triggers the measurements generation and raw data output.
#  - The navRate value defines that every nth measurement triggers a navigation 
#    epoch.
#  - The update rate has a direct influence on the power consumption. The more 
#    fixes that are required, the more CPU power and communication resources 
#    are required.
#  - For most applications a 1 Hz update rate would be sufficient.
#  - When using Power Save Mode, measurement and navigation rate can differ from
#    the values configured here

uint8 CLASS_ID = 6
uint8 MESSAGE_ID = 8

uint16 measRate           # Measurement Rate, GPS measurements are
                          # taken every measRate milliseconds [ms]
uint16 navRate            # Navigation Rate, in number of measurement
                          # cycles. On u-blox 5 and u-blox 6, this parameter
                          # cannot be changed, and always equals 1.
uint16 timeRef            # The time system to which measurements are aligned
uint16 TIME_REF_UTC = 0
uint16 TIME_REF_GPS = 1
uint16 TIME_REF_GLONASS = 2   # not supported in protocol versions less than 18
uint16 TIME_REF_BEIDOU = 3    # not supported in protocol versions less than 18
uint16 TIME_REF_GALILEO = 4   # not supported in protocol versions less than 18
